packages/react-components/src/components/TogetherModeOverlay.tsx

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(together-mode) */
+import React, { useMemo, useState } from 'react';
+/* @conditional-compile-remove(together-mode) */
+import {
+  Reaction,
+  ReactionResources,
+  VideoGalleryTogetherModeParticipantPosition,
+  VideoGalleryLocalParticipant,
+  VideoGalleryRemoteParticipant
+} from '../types';
+/* @conditional-compile-remove(together-mode) */
+import { moveAnimationStyles, spriteAnimationStyles } from './styles/ReactionOverlay.style';
+/* @conditional-compile-remove(together-mode) */
+import {
+  // getCombinedKey,
+  REACTION_NUMBER_OF_ANIMATION_FRAMES
+  // REACTION_START_DISPLAY_SIZE
+} from './VideoGallery/utils/reactionUtils';
+/* @conditional-compile-remove(together-mode) */
+import { Icon, Text } from '@fluentui/react';
+/* @conditional-compile-remove(together-mode) */
+import { getEmojiResource } from './VideoGallery/utils/videoGalleryLayoutUtils';
+/* @conditional-compile-remove(together-mode) */
+import { useLocale } from '../localization';
+/* @conditional-compile-remove(together-mode) */
+import { _HighContrastAwareIcon } from './HighContrastAwareIcon';
+/* @conditional-compile-remove(together-mode) */
+import {
+  calculateScaledSize,
+  getTogetherModeParticipantOverlayStyle,
+  setTogetherModeSeatPositionStyle,
+  togetherModeIconStyle,
+  togetherModeParticipantDisplayName,
+  togetherModeParticipantStatusContainer,
+  TogetherModeSeatStyle
+} from './styles/TogetherMode.styles';
+/* @conditional-compile-remove(together-mode) */
+import { CallingTheme, useTheme } from '../theming';
+/* @conditional-compile-remove(together-mode) */
+import { RaisedHandIcon } from './assets/RaisedHandIcon';
+/* @conditional-compile-remove(together-mode) */
+import { _pxToRem } from '@internal/acs-ui-common';
+/* @conditional-compile-remove(together-mode) */
+/**
+ * Signaling action overlay component props
+ *
+ * Can be used with {@link VideoTile}.
+ *
+ * @internal
+ */
+type TogetherModeParticipantStatus = {
+  reaction?: Reaction;
+  scaledSize?: number;
+  isHandRaised?: boolean;
+  isSpotlighted?: boolean;
+  isMuted?: boolean;
+  id: string;
+  seatPositionStyle: TogetherModeSeatStyle;
+  displayName: string;
+  showDisplayName: boolean;
+};
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * TogetherModeOverlay component renders an empty JSX element.
+ *
+ * @returns {JSX.Element} An empty JSX element.
+ */
+export const TogetherModeOverlay = React.memo(
```

Comment:
can we import `memo` from React here? or do we need to use the keyword? we want to avoid this if we can to make sure we are using the ESM module